_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Multi-tenancy aggregate view for saved objects
+parent: OpenSearch Dashboards multi-tenancy
+nav_order: 60
+---
+
+# Dashboards multi-tenancy aggregate view for saved objects
+
+Aggregate view for saved objects is an experimental feature for OpenSearch 2.4. Therefore, we do not recommend enabling the feature for a production environment at this time. For updates on the progress of aggregate view for saved objects, or if you'd like to leave feedback that could help improve the feature, see the [Dashboards object sharing](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2249) GitHub issue. For a more comprehensive view of the proposed future development of multi-tenancy, see the [Dashboards object sharing](https://github.com/opensearch-project/security/issues/1869) issue.
+{: .warning}
+
+Aggregate view for saved objects allows a user who has access to multiple tenants to see all saved objects associated with those tenants in a single view without having to switch between tenants to do so. This includes both tenants created by the user and tenants shared with the user. Aggregate view introduces a Tenants column in the Saved Objects table that gives the user the option to filter by tenants and make visible their associated saved objects.
+
+Once a user identifies a saved object of interest, the user can then switch to that tenant to work with the object.
```

Comment:
Global: Ensure that we're being consistent in our use of "you/your" versus "a user". I think "a user" is fine in the preceding paragraph, but I would probably try to use "you/your" from this point forward, as appropriate, because we're providing directions to the user.